sdk/elixir/lib/dagger/gen/container.ex

```diff
@@ -7,7 +7,7 @@ defmodule Dagger.Container do
   defstruct [:selection, :client]
 
   (
-    @doc "Initializes this container from a Dockerfile build.\n\n## Required Arguments\n\n* `context` - Directory context used by the Dockerfile.\n\n## Optional Arguments\n\n* `dockerfile` - Path to the Dockerfile to use.\n\nDefault: './Dockerfile'.\n* `build_args` - Additional build arguments.\n* `target` - Target build stage to build.\n* `secrets` - Secrets to pass to the build.\n\nThey will be mounted at /run/secrets/[secret-name]."
+    @doc "Initializes this container from a Dockerfile build.\n\n## Required Arguments\n\n* `context` - Directory context used by the Dockerfile.\n\n## Optional Arguments\n\n* `dockerfile` - Path to the Dockerfile to use.\n\nDefault: './Dockerfile'.\n* `build_args` - Additional build arguments.\n* `target` - Target build stage to build.\n* `secrets` - Secrets to pass to the build.\n\nThey will be mounted at /run/secrets/[secret-name] in the build container\n\nThey can be accessed in the Dockerfile using the \"secret\" mount type\nand mount path /run/secrets/[secret-name]\ne.g. RUN --mount=type=secret,id=my-secret curl url?token=$(cat /run/secrets/my-secret)\""
```

Comment:
@wingyplus For later, that would be awesome to improve doc formatting ;)

Reply:
Unfortunately, the codegen rely on Elixir AST, it cannot control the output. :(

The only way I can think at the moment is change code generation mechanism to use Dagger codegen which’s already started in https://github.com/dagger/dagger/pull/5597 but not complete yet. :)